broker/src/main/java/io/zeebe/broker/system/configuration/backpressure/Gradient2Cfg.java

```diff
@@ -10,20 +10,20 @@
 import static com.google.common.base.Preconditions.checkArgument;
 import static io.zeebe.broker.system.configuration.ConfigurationUtil.checkPositive;
 
-@SuppressWarnings("common-java:DuplicatedBlocks")
 public class Gradient2Cfg {
 
   private int minLimit = 10;
   private int initialLimit = 20;
   private double rttTolerance = 2.0;
   private int longWindow = 600;
 
+  @SuppressWarnings("common-java:DuplicatedBlocks")
```

Comment:
that does not work. Suppress warnings is a general annotation. To tell it to ignore sonar qube warnings, you would need
`@SuppressWarnings("squid:common-java:DuplicatedBlocks")` where _squid_ stands for SonarQube Unique ID, I guess.

Not sure whether you need to have _common-java_ in or not. When I used it, I didn't need it. I guess SonarQube knows it is looking at Java already. 

Alternatively, I also found:
> For instance, you can add the following property to your scanner configuration:
> sonar.cpd.exclusions=path/to/your/package/*.java
on this page: https://stackoverflow.com/questions/52865737/how-do-i-ignore-duplicated-code-report-in-sonar

I guess this could be set somewehere in the pom files, but I haven't done it yet.